addons/binding/org.openhab.binding.airquality/OSGI-INF/AirQualityDiscoveryService.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would be nice not to add any XMLs for DS components anymore, but rather refactor the existing services to use annotations (see e.g. [this change](https://github.com/eclipse/smarthome/pull/4068/files)).